tests/integration_tests.rs

```diff
@@ -1262,8 +1296,10 @@ async fn do_mutate(is_pessimistic: bool) -> Result<()> {
     }
 }
 
-async fn verify_mutate(is_pessimistic: bool) {
-    let client = TransactionClient::new(pd_addrs()).await.unwrap();
+async fn verify_mutate(is_pessimistic: bool) -> Result<()> {
+    let client =
+        TransactionClient::new_with_config(pd_addrs(), Config::default().with_default_keyspace())
+            .await?;
```

Comment:
It's LGTM here. But actually I prefer using `unwrap()` in test cases, as sometimes it's not easy to find out the error position when using `?`.

Reply:
I was just trying to keep the style with other tests.